src/frontend/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "tsc --noEmit && webpack",
     "build-dev": "yarn copy-iframe-resizer; yarn build",
+    "clean": "find ../backend/marsha/static/js/ -type f ! -iname 'iframeResizer.min.js' -delete",
```

Comment:
I find it would be much clearer like this (even if performs more operations), WDYT ?
```suggestion
    "clean": "rm -r ../backend/marsha/static/js/ && yarn copy-iframe-resizer",
```

Reply:
`rm` can not be used. It fails because there is too many files to delete.

Reply:
hmm ok, I see…